test/NuGet.Core.Tests/NuGet.Common.Test/RuntimeEnvironmentHelperTests.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    public class RuntimeEnvironmentHelperTests
+    {
+        [Fact]
+        public void RuntimeEnvironmentHelper_MutuallyExclusiveRuntimes()
+        {
+            // It only should return true in either Windows or MacOS/Linux
```

Comment:
Remove the comment.  It explains what the expected result is.

Comments should explain _why_, which is important when underlying reasons are unclear from the code itself.

It should be clear from reading the assertion what the expected result.